src/containers/Sidebar/index.js

```diff
@@ -92,18 +92,18 @@ const Sidebar = (props) => {
 						</div>
 					}
 					<div className="sidebar-nav">
-
 						<Nav  vertical>
 							{memoizedItemsList.map((item, idx) => (
 								<SidebarItem
 									key={idx}
 									item={item}
 									unauthorizedNavChildren={unauthorizedNavChildren}
 									uncollapseAll={memoizedItemsList.length <= 2}
+									toggleSidebar={toggle}
```

Comment:
@Aringoaway Can we unify naming and instead of `toggle` use `toggleSidebarModal`?

```
toggleSidebarModal={toggleSidebarModal}
```